Doc/library/ctypes.rst

```diff
@@ -308,12 +308,12 @@ bytes objects are immutable)::
 
    >>> s = "Hello, World"
    >>> c_s = c_wchar_p(s)
-   >>> print(c_s)
-   c_wchar_p('Hello, World')
+   >>> print(c_s, c_s.value)
```

Comment:
If you really want to keep `print(c_s)`, I'd say move `c_s.value` to a new line:

```py
>>> print(c_s)
c_wchar_p(139966785747344)
>>> print(c_s.value)
Hello World
```